mne/io/edf/tests/test_edf.py

```diff
@@ -385,3 +386,10 @@ def test_edf_lowpass_zero():
     with pytest.warns(RuntimeWarning, match='too long.*truncated'):
         raw = read_raw_edf(edf_stim_resamp_path)
     assert_allclose(raw.info["lowpass"], raw.info["sfreq"] / 2)
+
+
+@testing.requires_testing_data
+def test_edf_annot_ms_onset():
+    """Test reading of sub-milisecond annotation onsets."""
```

Comment:
To be exact, this should be "sub-second annotation onsets". Accordingly, the name for the test data should be `edf_annot_sub_s_path`.